src/libs/API/types.ts

```diff
@@ -288,6 +288,7 @@ const WRITE_COMMANDS = {
     ADD_BILLING_CARD_AND_REQUEST_WORKSPACE_OWNER_CHANGE: 'AddBillingCardAndRequestPolicyOwnerChange',
     SET_POLICY_DISTANCE_RATES_UNIT: 'SetPolicyDistanceRatesUnit',
     SET_POLICY_DISTANCE_RATES_DEFAULT_CATEGORY: 'SetPolicyDistanceRatesDefaultCategory',
+    SET_POLICY_PER_DIEM_RATES_DEFAULT_CATEGORY: 'SetPolicyPerDiemRatesDefaultCategory',
```

Comment:
This tests well, but we now have a more efficient command in the backend:
`SetCustomUnitDefaultCategory (AuthToken, policyID, customUnitID, category)`. Can you use that one?

Reply:
Ah, maybe. I'll try to test today

Reply:
Actually, how do you unset it? I can't seem to do it in the frontEnd